packages/web-react/src/components/Alert/README.md

```diff
@@ -44,6 +44,9 @@ and [escape hatches][readme-escape-hatches].
 
 For detailed information see [Alert][alert] component.
 
+Please pay attention to the accessibility setting when Alert is dynamically displayed. In cases where you need to use the component as an information that requires the user's immediate attention,
```

Comment:
Maybe we can an emoji here? So it is more noticeable? But I will leave it up to you.

Reply:
I added an icon. I hope I chose it correctly